lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/LibertyWorkspace.java

```diff
@@ -206,4 +210,24 @@ public String toString() {
         return workspaceFolderURI;
     }
 
+    public void setFeatureListGraph(FeatureListGraph featureListGraph) {
+        this.featureListGraph = featureListGraph;
+        if (isLibertyInstalled) {
+            this.featureListGraph.setRuntime(libertyRuntime + "-" + libertyVersion);
+        }
+    }
+
+    public FeatureListGraph getFeatureListGraph() {
+        String workspaceRuntime = libertyRuntime + "-" + libertyVersion;
+        boolean generateGraph = featureListGraph.isEmpty() || !featureListGraph.getRuntime().equals(workspaceRuntime);
+        if (this.isLibertyInstalled && generateGraph) {
```

Comment:
There is a method in this class that clears out the installed feature list when liberty is not installed. That method should also clear out the feature list graph.